src/newfight.hpp

```diff
@@ -197,9 +197,11 @@ struct melee_combat_data {
     if (weapon) {
       if (ranged_combat_mode) {
         if (does_weapon_have_bayonet(weapon)) {
+          power += 2; // Cannon Companion page 33 table, bayonet is listed as +2 Power (or errata if misprinted) -Vile
           dam_type = TYPE_PIERCE;
           skill = SKILL_POLE_ARMS;
         } else {
+          power += 2; // Cannon Companion page 11 table, rifle-butt is listed as +2 Power. -Vile
```

Comment:
What about pistol butt etc? This applies to all ranged weapons if it's done here. If they're all supposed to have +2, that's fine.

Reply:
Pistol is supposed to only be +1 in comparison